src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.ts

```diff
@@ -209,8 +209,8 @@ export class UserFormComponent implements OnInit {
         } else {
           this.notificationService.show(
             NotificationType.success,
-            `User "${userFormModel.username}" has been updated.`,
-            'Edit User'
+            'Edit user "${userFormModel.username}"',
+            `User "${userFormModel.username}" has been updated.`
```

Comment:
This argument is now only used for error message descriptions. Please remove this line.